frontend/src/components/CodebookLabelMenu/index.jsx

```diff
@@ -97,13 +97,14 @@ class CodebookLabelMenu extends React.Component {
         return (
             <div className="margin-bottom-15 no-overflow">
                 <div className="row" id="label_group_buttons">
-                    <ButtonGroup className="pull-left">
-                        {label_button}
-                        {codebook_button}
-                    </ButtonGroup>
+                    {CODEBOOK_URL != "" ? (
+                        <ButtonGroup className="pull-left">
+                            {label_button}
+                            {codebook_button}
+                        </ButtonGroup>
+                    ) : label_button}
                 </div>
                 {this.getLabels(labels, this.state.labels_open)}
-                {codebook_module}
```

Comment:
I'm not getting the codebook to pop up when I click the button. I'm on Chrome. Have you had this issue?

Reply:
I'm not even sure how/why the lines for the codebook module got removed but I've added them back in and things seem to be working now.